packages/@angular/cli/utilities/app-utils.ts

```diff
@@ -0,0 +1,11 @@
+export function getAppFromConfig(apps: any, name: String) {
```

Comment:
I changed the whole testing config setup in the last day so it should be much more like the others.